ethpm/tools/builder.py

```diff
@@ -373,13 +373,13 @@ def normalize_contract_type(
 
 @to_dict
 def normalize_bytecode_object(obj: Dict[str, Any]) -> Iterable[Tuple[str, Any]]:
-    if obj["linkReferences"]:
-        yield "link_references", process_link_references(
-            obj["linkReferences"], obj["object"]
-        )
-        yield "bytecode", process_bytecode(obj["linkReferences"], obj["object"])
+    link_references = obj.get("linkReferences")
```

Comment:
We try to avoid return value checking due to it being error prone (it is easy to forget to make the check).  Maybe use a `try/except` pattern instead?